remill/Arch/X86/Runtime/State.h

```diff
@@ -566,9 +564,11 @@ struct alignas(16) State final : public ArchState {
   MMX mmx;  // 128 bytes.
   FPUStatusFlags sw;  // 8 bytes
   XCR0 xcr0;  // 8 bytes.
+  FPUControlWord fpu_control;  // 2 bytes;
+  uint8_t _0[14];  // 14 bytes.
```

Comment:
Everything looks great; my only side comment is about the padding -- is there any way to make it a macro (e.g. `PAD_16`) or something to show that its there to pad things out to a 16 byte boundary?

Reply:
would rewriting it as
```c++
uint8_t _padding[14];  // Pad to a 16-byte boundary.
```
be acceptable?

Reply:
Absolutely